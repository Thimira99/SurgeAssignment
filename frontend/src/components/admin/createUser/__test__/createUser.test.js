import React from 'react';

import ReactDOM from 'react-dom';

import CreateUser from '../createUser';

import { BrowserRouter } from 'react-router-dom';

import { cleanup } from '@testing-library/react';

import "@testing-library/jest-dom/extend-expect";



afterEach(cleanup);

it("renders without crashing", () => {

    const div = document.createElement("div");

    ReactDOM.render(

        <BrowserRouter>

            <CreateUser></CreateUser>

        </BrowserRouter>, div)

})

