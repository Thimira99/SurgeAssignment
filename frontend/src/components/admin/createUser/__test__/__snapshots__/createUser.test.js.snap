// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  data-testId="user"
>
  <nav
    class="navbar navbar-expand-lg navbar-light bg-light"
    id="mainNav"
  >
    <a
      class="navbar-brand"
      href="#"
    >
      Admin
    </a>
    <button
      aria-controls="navbarNavAltMarkup"
      aria-expanded="false"
      aria-label="Toggle navigation"
      class="navbar-toggler"
      data-target="#navbarNavAltMarkup"
      data-toggle="collapse"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="collapse navbar-collapse"
    >
      <div
        class="navbar-nav"
        id="navbarNavAltMarkup"
      >
        <a
          class="nav-item nav-link active"
          href="/getUsers"
        >
          User List
        </a>
        <a
          class="nav-item nav-link"
          href="/createUser"
        >
          Create User
        </a>
        <div
          className="logout"
        >
          <span
            class="nav-item nav-link"
          >
            Hello 
          </span>
          <button
            className="btn btn-outline-danger"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": "1rem",
              }
            }
          >
            Logout
          </button>
        </div>
      </div>
    </div>
  </nav>
  <div
    className="form_container"
  >
    <form
      className="form"
      onSubmit={[Function]}
    >
      <h1>
        Create Account
      </h1>
      <input
        className="input"
        name="firstName"
        onChange={[Function]}
        placeholder="First Name"
        required={true}
        type="text"
      />
      <input
        className="input"
        name="lastName"
        onChange={[Function]}
        placeholder="Last Name"
        required={true}
        type="text"
      />
      <input
        className="input"
        name="email"
        onChange={[Function]}
        placeholder="Email"
        required={true}
        type="email"
      />
      <input
        className="input"
        name="date"
        onChange={[Function]}
        placeholder="Date Of Birth"
        required={true}
        type="date"
      />
      <input
        className="input"
        name="mobile"
        onChange={[Function]}
        placeholder="Mobile Number"
        required={true}
        type="text"
      />
      <input
        className="input"
        name="mobile"
        onChange={[Function]}
        placeholder="Account Type"
        required={true}
        type="text"
      />
      <button
        className="greenBtn"
        type="submit"
      >
        Create
      </button>
    </form>
  </div>
</div>
`;
